calico/datamodel_v1.py

```diff
@@ -32,13 +32,19 @@
 # All Calico data is stored under this path.
 ROOT_DIR = "/calico"
 
+# Current version
+VERSION = "/v1"
```

Comment:
I don't think I agree with this change.  It makes it look like the version of the two subtrees should be incremented together, which is not necessarily the case.  We may want to rev the felix version without revving the openstack version, for example.